Update playing videos (#10614)
godotengine/godot-docs: tutorials/animation/playing_videos.rst

Supported playback formats
--------------------------

The only supported format in core is **Ogg Theora** (not to be confused with

> Ogg Vorbis audio) with optional Ogg Vorbis audio tracks. It's possible for
> extensions to bring support for additional formats, but no such extensions
> exist yet as of July 2022.

Reviewer: If this is still true, we should update the "as of" (the date will likely be different again by the time that the PR gets reviewed:
```suggestion
Ogg Vorbis audio) with optional Ogg Vorbis audio tracks. It's possible for
extensions to bring support for additional formats, but no such extensions
exist yet as of February 2025.
```
If this is false, or we can't confirm it easily, let's just remove the statement:
```suggestion
Ogg Vorbis audio) with optional Ogg Vorbis audio tracks. It's possible for
extensions to bring support for additional formats.

   You can check this by running ``ffmpeg`` without any arguments, then looking
   at the ``configuration:`` line in the command output.

.. warning::

>    FFmpeg will erroneuously drop identical frames when performing a copy of a

Reviewer 2: ```suggestion
   FFmpeg will erroneously drop identical frames when performing a copy of a
```

   You can check this by running ``ffmpeg`` without any arguments, then looking
   at the ``configuration:`` line in the command output.

.. warning::

   All FFmpeg releases before Feb 20th, 2025 could produce bad video streams
   due to a couple of bugs. It's highly recommended to use one of the latest
   static daily builds, or build FFmpeg from their master branch where they're
   already fixed. More details in `FFmpeg issue #11451
   <https://trac.ffmpeg.org/ticket/11451>`__ and `FFmpeg issue #11454
   <https://trac.ffmpeg.org/ticket/11454>`__.

> 

Author: I was hesitant to include the reference to the bugs and only did it because there is no stable release. I'll remove it.

The warning to use the latest builds I think it's necessary to avoid reports of videos that don't play correctly. Would you remove that too?

> .. warning::
> 
>    All FFmpeg releases before Feb 20th, 2025 could produce bad video streams
>    due to a couple of bugs. It's highly recommended to use one of the latest
>    static daily builds, or build FFmpeg from their master branch where they're
>    already fixed. More details in `FFmpeg issue #11451
>    <https://trac.ffmpeg.org/ticket/11451>`__ and `FFmpeg issue #11454
>    <https://trac.ffmpeg.org/ticket/11454>`__.
> 

Reviewer: I'm not that familiar with either FFmpeg's release cycle or these specific bugs. From a brief look at their release policy, these bugfixes are guaranteed to be included in their next major version, and may or may not be included in the next stable minor versions of FFmpeg?

If that is the case, I think it's okay to include a warning in our docs, *until* there is a stable FFmpeg version with the bugfixes included. Which probably means removing the warning in the 4.5 or 4.6 version of our docs.

I'd suggest including this UPDATE comment above the warning, so the note can be easily removed once it's no longer needed:
```suggestion
.. UPDATE: When the FFmpeg bugfixes for https://trac.ffmpeg.org/ticket/11451 and
.. https://trac.ffmpeg.org/ticket/11454 are included in a stable FFmpeg release,
.. this note can be removed. That will likely be FFmpeg 8.0, and will likely
.. happen during the Godot 4.5 or 4.6 release cycle.

.. warning::

   All FFmpeg releases before Feb 20th, 2025 could produce bad video streams
   due to a couple of bugs. It's highly recommended to use one of the latest
   static daily builds, or build FFmpeg from their master branch where they're
   already fixed. More details in `FFmpeg issue #11451
   <https://trac.ffmpeg.org/ticket/11451>`__ and `FFmpeg issue #11454
   <https://trac.ffmpeg.org/ticket/11454>`__.

```

for a table listing Ogg Vorbis audio quality presets and their respective
variable bitrates.

> The **GOP size** (``-g:v``) can provide further compression by increasing the

Reviewer: ```suggestion
The group of pictures size or **GOP size** (``-g:v``) can provide
further compression by increasing the
```
Might be worth writing out "group of pictures" at least once when introducing this? It's not a term I was familar with, at least. We do the same for much more common terms like GPU elsewhere in the docs.

interval between keyframes. The default value is pretty low (12). Higher values
can provide better compression at the expense of some quality and slightly
slower seeks. Increasing GOP size usually produces better results than reducing
video quality and it's thus recommended.

.. note::

>    GOP sizes above 64 will cause seeks to be slower, more noticeably beyond
>    every power of two. Also, due to some supposed bug in FFmpeg, using a GOP
>    size greater than 64 can slow down seeking even more and might create bigger
>    files.

Reviewer: ```suggestion
   GOP sizes above ``64`` will cause seeks to be slower, more noticeably beyond
   every power of two. Also, due to some supposed bug in FFmpeg, using a GOP
   size greater than ``64`` can slow down seeking even more and might create bigger
   files.
```
Use `code` for literal values.

Also, if there is a bug report we can reference for the FFmpeg bug we should link to it.
